sycl/include/CL/sycl/detail/kernel_desc.hpp

```diff
@@ -13,6 +13,8 @@
 #include <CL/sycl/detail/defines_elementary.hpp>
 #include <CL/sycl/detail/export.hpp>
 
+#include <cstdint>
```

Comment:
Nice, perhaps this will also fix one of internal trackers reporting problems with undefined int8_t type slipped into integration-header file.

Reply:
This inclusion is giving oneapi/dpl a stomach ache.

Reply:
because of this: https://github.com/steffenlarsen/llvm/blob/8a1c97c5705cd904795ce0e5a8856ac0e9ac7010/sycl/include/CL/sycl/detail/kernel_desc.hpp#L11